src/main/java/com/example/kwbruunauktion/auktionSystem/service/users/UserAdminService.java

```diff
@@ -0,0 +1,84 @@
+package com.example.kwbruunauktion.auktionSystem.service.users;
+
+import com.example.kwbruunauktion.auktionSystem.dto.users.request.UserAdminRequest;
+import com.example.kwbruunauktion.auktionSystem.dto.users.response.UserAdminResponse;
+import com.example.kwbruunauktion.auktionSystem.entity.users.UserAdmin;
+import com.example.kwbruunauktion.auktionSystem.repository.users.UserAdminRepository;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+
+@Service
+public class UserAdminService {
+  private final UserAdminRepository userAdminRepository;
+
+  public UserAdminService(UserAdminRepository userAdminRepository) {
+    this.userAdminRepository = userAdminRepository;
+  }
+
+  public List<UserAdminResponse> getAllUserAdmin() {
+    List<UserAdmin> allUserAdminsList = userAdminRepository.findAll();
+    return allUserAdminsList.stream().map(UserAdminResponse::new).toList();
+
+  }
+
+  public List<UserAdminResponse> getAllUserAdmin(Pageable p) {
+    Page<UserAdmin> allUserAdminsList = userAdminRepository.findAll(p);
+    return allUserAdminsList.stream().map(UserAdminResponse::new).toList();
+
+  }
+
+  public UserAdminResponse getUserAdminById(Long id) {
+    UserAdmin foundUserAdmin = userAdminRepository.findById(id).orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND, "User not found"));
+    return new UserAdminResponse(foundUserAdmin);
+  }
+
+  public void addUserAdmin(UserAdminRequest userAdminRequest) {
+    if (userAdminRepository.existsById(userAdminRequest.getId())) {
+      throw new ResponseStatusException(HttpStatus.BAD_REQUEST,"User with this ID already exist");
+    }if (userAdminRepository.existsByUsername(userAdminRequest.getUserName())) {
+      throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "User with this username already exist");
+    }
+    UserAdmin newUserAdmin = UserAdminRequest.getUserAdminEntity(userAdminRequest);
+    newUserAdmin = userAdminRepository.save(newUserAdmin);
+    new UserAdminResponse(newUserAdmin);
```

Comment:
Samme som før her. De ny UserAdminResponse bilver ikke brugt til noget.

Reply:
Har lagt den til, hvis vi vælger i frontend at lave en form for bekræftelse at den her bruger er oprettet skal man bare ligge en return på :D